### DISCUSSION TITLE
Relay Connections - does it induce lag?

### DISCUSSION THREAD

**Discussion by GTMoraes** [YimMenu/YimMenu#1914] [Q&A] [answered]
Just wondering whether leaving Relay Connections on causes lag. 
Last PVPs I had, people were complaining I was teleporting around, and indeed it was too easy to defeat them, as they weren't really shooting at me unless I was walking in a straight line.

Was it just a bad connection, or does Relay Connection protection induces lag?

**Comment by Yimura** [comment, marked as answer]
Relays may introduce extra latency on the connection and other people may observe you to be laggy.
